float64 x
float64 x_dot
float64 theta
float64 theta_dot
float64 psi
float64 psi_dot